### PR TITLE
[COM-131] Implementation of Badge component

### PR DESCRIPTION
**Pull request by Angele91**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)

https://impulsumvc.atlassian.net/browse/COM-131

**Storybook Link** (Link to a local storybook url to review new functionality)

http://localhost:8080/?path=/story/components-primitives-badge--all

### CODE REVIEW

File: packages/components/src/theme/ComposerTheme/Components/index.ts [hunk 1]
```diff
@@ -3,11 +3,13 @@ import { Spinner } from './Spinner';
...
 import { TextPairing } from './TextPairing';
+import { Badge } from './Badge';
 
 export const components = {
   Button,
   Spinner,
   Tabs,
   Text,
   TextPairing,
+  Badge,
```
Comment (tkanzakic) [inline packages/components/src/theme/ComposerTheme/Components/index.ts:14]: please follow an alphabetical order (yes, we should automate this 😉)

File: packages/components/src/components/primitives/Badge/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from './Badge';
```
Comment (tkanzakic) [inline packages/components/src/components/primitives/Badge/index.ts:1]: missing re-export at the parent folder index.ts in alphabetical order

**Review by tkanzakic — COMMENTED** [review]
Great work!, just a couple of minor comments

File: packages/components/src/theme/ComposerTheme/Components/index.ts (hunk 1 above)
Comment (Angele91) [reply packages/components/src/theme/ComposerTheme/Components/index.ts]: Done!

**Review by tkanzakic — APPROVED** [review]
LGTM!

**Pull request merged**